soco/exceptions.py

```diff
@@ -80,3 +80,37 @@ class SoCoSlaveException(SoCoException):
 
 class NotSupportedException(SoCoException):
     """Raised when something is not supported by the device"""
+
+
+class SoCoFault(object):
+    """Class to represent a failed object instantiation.
+
+    It rethrows the exception on common use.
```

Comment:
If we don't do aything else, I think we should document the exception property explicitely, I think the name of the Sphinx/Napoleon section is simply attribute.

Reply:
Good idea.

Reply:
As above